spec/factories/teacher_factory.rb

```diff
@@ -8,6 +8,19 @@
       corrected_name { [trs_first_name, Faker::Name.middle_name, trs_last_name].join(' ') }
     end
 
+    trait :with_sparsity_uplift do
+      sparsity_uplift { true }
+    end
+
+    trait :with_pupil_premium_uplift do
+      pupil_premium_uplift { true }
+    end
+
+    trait :with_uplift do
```

Comment:
Hi @mooktakim do we need these traits as we're still not using anywhere? or do you think it's good to add anyway? tks

Reply:
Good question. I suppose because its related to Teacher, its good that its there as it'll show as one of the options.

Reply:
We've been adding the factories/traits for other new fields, FWIW